Makefile

```diff
@@ -1,2 +1,5 @@
+exec:
```

Comment:
I think it would be more idiomatic if we use `run`

Reply:
I'm not very familiar with idiomatic way of doing this - but just to confirm: the `exec` command I added doesn't really run anything, it just complies the executable.